admin_manual/maintenance/upgrade.rst

```diff
@@ -91,3 +91,22 @@ Change ``"maintenance" => false`` to ``"maintenance" => true``:
     "maintenance" => true,
 
 Then change it back to ``false`` when you are finished.
+
+Manual steps during upgrade
+---------------------------
+
+Some operation can be quite time consuming. Therefore we decided not to add them
+the the normal upgrade process. We recommend to run them manually after the upgrade
+was completed. Below you find a list of this commands.
+
+Upgrading to Nextcloud 13
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With Nextcloud 13 we added a new index to the share table which should result in
+significant performance improvements:
+
+ $ sudo -u www-data php occ db:add-missing-indice
+
+With Nextcloud 13 we switched to bigint for the file ID's in the file cache table
```

Comment:
same here

Reply:
Here we also don't use `::` https://github.com/nextcloud/documentation/pull/671/files#diff-368ad62123233efc28400a735b869cf0L82
indention should be enough, preview looks fine too.